modules/lib/src/main/resources/assets/js/app/wizard/ContentWizardPanel.ts

```diff
@@ -1234,7 +1229,24 @@ export class ContentWizardPanel
         this.settingsWizardStep = new ContentWizardStep(i18n('field.settings'), this.settingsWizardStepForm, 'icon-wrench');
         steps.push(this.settingsWizardStep);
 
-        return steps;
+        const editPermissionsIcon = this.canEveryoneRead(this.getPersistedItem()) ? 'icon-unlock' : 'icon-lock';
```

Comment:
1) `editPermissionsIconClass` or just `iconClass` would be more self-describing name for a variable
2) Type is missing